### ISSUE TITLE
Forced override setting for other dimensions than DIM 100

### ISSUE THREAD

**Issue by AlteOgre** [VulcanForge/pvp-mode#249]
I suspect we have overseen this one. For now, I believe the global PvP Mode condition is valid in all worlds.
It would be helpful and sufficient imo if players would have three (server determined) options in other worlds (Overworld, Nether, End and Utumno (DIM 101, biome id 0)):
* Free to choose: PvP Toggling enabled.
* Get PvP Mode forced ON.
* Get PvP Mode forced OFF.
Obviously, this would require configuration entries. I think a separate file would be fitting, in line with latest insights.

This could also address compatibility with other mods that add dimensions to the game, like Aether and Twilight Forest. It may be wisest to address those in separate issues, so we can tackle those later, after this one.

**Comment by CraftedMods** [comment]
Required a major version, probably.

**Comment by DwarfyAssassin** [comment]
Would it also be possible to make specific regions from world guard have set rules. Like in the world guard called "arena" pvp would be forced on. 
Maybe you could also add more config options for where partial inventory drop is active. For example it would be disabled in the overworld and enabled in middle earth.

**Comment by AlteOgre** [comment]
Yes, Worldguard compatibility can be considered a planned extension of this feature.
